data/org_cluster_map_handler.py

```diff
@@ -0,0 +1,138 @@
+                ' \'getorg\' package.')
+
+    # For creating the organization map, the 'getorg' uses a 'Nominatim' named
+    # package which geocodes the contributor location and then uses that class
+    # to create the map. Since, we're not dealing with that function which use
+    # that 'Nominatim' package because we're fetching a JSON data and storing
+    # it in our db. Therefore, defining our own simple class that can aid us
+    # to create a cluster map.
+    class Location:
+
+        def __init__(self, longitude, latitude):
+            self.longitude = longitude
+            self.latitude = latitude
+
+    org_location_dict = {}
+
+    for contrib in Contributor.objects.all():
+        if not contrib.login.startswith('testuser') and contrib.location:
+            user_location = json.loads(contrib.location)
+            location = Location(user_location['longitude'],
+                                user_location['latitude'])
+            org_location_dict[contrib.login] = location
+            logger.debug('{} location {} added on map'.format(
+                contrib.login, user_location))
+    getorg.orgmap.output_html_cluster_map(org_location_dict,
+                                          folder_name=output_dir)
+
+    move_and_make_changes_in_files(output_dir)
+
+
+def move_and_make_changes_in_files(output_dir):
+    """
+    # Move static files from 'output_dir' to django static folder and
+    # the 'map.html' file to django templates directory to get it displayed
+    # on the homepage with the needed django syntax and CSS in the html file.
+    :param output_dir: Directory from where the files have to be moved
+    :return: None
+    """
+
+    # Move leaflet_dist folder to static folder
+    leaflet_source_path = '{}/{}/leaflet_dist/'.format(os.getcwd(),
+                                                       output_dir)
+    leaflet_destination_path = '{}/{}/leaflet_dist/'.format(os.getcwd(),
+                                                            'static')
+
+    # Remove existing leaflet_dir if exists
+    for root, dirs, files in os.walk(leaflet_destination_path):
+        for file in files:
+            os.remove(os.path.join(leaflet_destination_path, file))
+        os.rmdir(root)
+
+    os.renames(leaflet_source_path, leaflet_destination_path)
+
+    # Move org_locations.js to static folder
+    locations_source_path = '{}/{}/org-locations.js'.format(os.getcwd(),
+                                                            output_dir)
+    locations_destination_path = '{}/{}/org-locations.js'.format(
+        os.getcwd(), 'static')
+    os.rename(locations_source_path, locations_destination_path)
+
+    # Make changes in map.html to support django syntax with needed CSS
+    with open('{}/map.html'.format(output_dir)) as f:
+        django_supported_htmls = []
+        lines = f.readlines()
+        for index in range(len(lines)):
+            line = lines[index].strip()
```

Comment:
I guess, you can directly do:
```python
In [1]: with open('file.txt') as f:
   ...:     for line in f:
   ...:         print (line)
```
see https://stackoverflow.com/questions/39086/search-and-replace-a-line-in-a-file-in-python